src/components/ui/ManagementUI.tsx

```
@@ -408,9 +429,88 @@ const ManagementUI = () => {
                         >
                             TEST MODE
                         </button>
+                        <button
+                            onClick={showTutorialWindow}
+                            className={`${styles.actionButton} ${styles.tutorialButton}`}
+                            title="Open Tutorial"
+                        >
+                            ?
+                        </button>
+                        <button
+                            onClick={() => (window as any).toggleFlowWindow?.()}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)